index.bs

```diff
@@ -1563,6 +1563,37 @@ Stats {#streaming-stats}
 TODO
 
 
+Protocol Extensions {#protocol-extensions}
+===================
+
+Open Screen agents may exchange extension messages that are not defined by this
+specification.  This could be used for experimentation, customization or other
+purposes.
+
+If an agent wishes to support extension messages, the author of the extension
+must register a capability ID and a range of message type keys in a public
+registry, whose location will be determined later.  Agents that accept the
+extension messages should include the corresponding capability ID in the
+`capabilities` field of its `agent-info` message.
+
+Capability IDs 1-999 are reserved for use by the Open Screen Protocol.
+Capability IDs 1000 and above are available for extensions.  See [[#appendix-b]]
+for legal ranges for extension message type keys.
+
+Note: This will prevent conflicts between multiple extension authors' capability
```

Comment:
Nit: "This" is too vague here. 

Maybe: 

Note: The purpose of the public registry is to prevent conflicts between multiple extension authors' capability IDs and message type keys.

Reply:
Done.